### PR TITLE
Dev containerservice microsoft.container service 2019 11 01

### PR DESCRIPTION
**Pull request by gtracer**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-11-01/managedClusters.json [hunk 1]
```diff
@@ -1619,6 +1619,19 @@
           "default": "172.17.0.1/16",
           "description": "A CIDR notation IP range assigned to the Docker bridge network. It must not overlap with any Subnet IP ranges or the Kubernetes service address range."
         },
+        "OutboundType": {
```
Comment (ryansbenson) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-11-01/managedClusters.json]: Adding these properties classifies as a breaking api change and a new api-version is required:
https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Breaking%20changes%20guidelines.md#new-property-added-to-response

Comment (gtracer) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-11-01/managedClusters.json]: @ryansbenson this is a new api-version. We are just a few weeks late on releasing `2019-11-01`

Comment (ryansbenson) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-11-01/managedClusters.json]: I see, this isn't a merge to master.

**Review by ryansbenson — APPROVED** [review]
Signed off from ARMs side

**Comment by robbiezhang** [comment]
LGTM

**Review by robbiezhang — APPROVED** [review]

**Review by zikalino — APPROVED** [review]

**Pull request merged**